### ISSUE TITLE
Get a better certificate for etrecheck.com

### ISSUE THREAD

**Issue by etresoft** [etresoft/EtreCheck#74]
The Let's Encrypt certificate doesn't seem to work for anything before 10.10

**Comment by jkbullard** [comment]
This may be related to #73.

I had what I believe was a similar problem when I tightened the security of the [Tunnelblick website](https://tunnelblick.net). I don't recall exactly, but I think it was requiring https: and TLS 1.2 that caused Safari on OS X 10.6.8 (and perhaps other versions of OS X) to not be able to negotiate a secure connection. (Firefox on 10.6.8 worked fine, presumably because it does it's own SSL negotiation). The Tunnelblick application's Sparkle (using the OS SSL) also could not could not successfully connect to its appcast when accessed when the application was running on 10.6.8.

In the end, and because as far as I know it only affected 10.6.8, I decided to keep the tightened security settings, and, essentially, abandon 10.6.8.

**Comment by etresoft** [comment]
Thanks for replying. I went back and double-checked and found something interesting.

This is definitely the same issue as #73. That issue was just to document a change to re-define "secure" to be "http" on older OS versions. I don't have any sensitive data so it is all just a warm fuzzy anyway. This issue is to consider getting a fancier certificate so I don't have this problem anymore.

But after your comment, I went back and tested. I cannot reproduce the problem today. A couple of days ago, my secure connections were definitely failing in 10.6-10.9 and succeeding in 10.11 and 10.12. Today I tried it on 10.10 and it worked fine there too. But today it also works on everything > 10.6. It was definitely broken 2 days ago and maybe before that. I had noticed an increase in people e-mailing reports to me instead of using the automated services (which are linked to an https protocol). I think it may have been broken for at least a few days. Since the OS in my VMs hasn't changed, it must have been something on the Let's Encrypt side. 

Now our problems are the same, just 10.6. I'm not ready to abandon 10.6 just yet. But I definitely think I need a better certificate. I don't mind using Let's Encrypt for experiments, but it shouldn't be in my production stream like it is.
